### PR TITLE
Add theme gruvbox.

### PR DESCRIPTION
**Pull request by SophrTemin**

### CODE REVIEW

**Review by FarParticul — COMMENTED** [review]
review

File: themes/Gruvbox/Readme.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+SPDX-FileCopyrightText: 2024 FarParticul
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+Gruvbox
+---
+[Gruvbox](https://github.com/morhetz/gruvbox) style theme for fans of this palette
+
+## Preview
+![Grvbx Preview](preview.png)
```
Comment (TheKodeToad) [inline themes/Gruvbox/Readme.md:11]:
```suggestion
![Gruvbox Preview](preview.png)
```

File: themes/Gruvbox/resources/.keep
Comment (TheKodeToad) [inline themes/Gruvbox/resources/.keep:1]: it's a bit excessive to add a license to an empty file

**Review by TheKodeToad — CHANGES_REQUESTED** [review]
Sorry for the long waiting time! I was going to review but I got distracted.

File: themes/Gruvbox/resources/.keep
Comment (SophrTemin) [reply themes/Gruvbox/resources/.keep:1]: If you need to remove it, I'll remove it, after all I copied from others)

**Comment by SophrTemin** [comment]
I'll probably write here, but it turned out that when I posted the icons I did not upload the catalog with them and it turned out that I lost them (request with icons but without icons turns out).

**Comment by SophrTemin** [comment]
can I change the sdpx names to sophrtemin now? I just happened to have changed my nickname all this time

File: themes/Gruvbox/resources/.keep
Comment (TheKodeToad) [reply themes/Gruvbox/resources/.keep:1]: It's not in other themes, I don't think

Comment (SophrTemin) [reply themes/Gruvbox/resources/.keep:1]: > It's not in other themes, I don't think

I thought it was, I will remove it in another request and close this one (there is already a theme and icons).

**Pull request closed without merging**